### PR TITLE
unit testing for placing mulitple buildings done.

### PR DESCRIPTION
**Pull request by Harish110**

### CODE REVIEW

**Comment by kenwongz** [comment]
@Harish110 this also include the scoring for shop right

File: SimpCityTests/GameTests.cs [hunk 1]
```diff
@@ -28,6 +28,134 @@ public class GameTests {
             }));
         }
 
+        /// <summary>
+        /// QA-SN-5, US-6:
```
Comment (CaeNGnp) [inline SimpCityTests/GameTests.cs]: should remove, see 750a4f5a26587975e4e5b4e9c54e33c01befb67e

**Review by CaeNGnp — CHANGES_REQUESTED** [review]
The added tests scripts for each building types are way too trivial to require their own test method. This unnecessarily adds burden for QA and developers when analysing test results.

Please combine all of the testing of house/factory/highway/shop/etc in one single test method (e.g. `BuildAt_PlacesAllBuildingTypes_WhenCalledProperly`) - we have more than enough cell spaces for these.

**Comment by Harish110** [comment]
> @Harish110 this also include the scoring for shop right

Yes @kenwongz

File: SimpCityTests/GameTests.cs [hunk 2]
```diff
@@ -18,16 +18,23 @@ public class GameTests {
 
             // Build a beach at 0,1
             game.BuildAt(game.buildingInfo[BuildingTypes.Beach], new CityGridPosition(0, 1));
+            game.BuildAt(game.buildingInfo[BuildingTypes.Factory], new CityGridPosition(1, 1));
+            game.BuildAt(game.buildingInfo[BuildingTypes.Highway], new CityGridPosition(0, 0));
+            game.BuildAt(game.buildingInfo[BuildingTypes.House], new CityGridPosition(1, 0));
+            game.BuildAt(game.buildingInfo[BuildingTypes.Monument], new CityGridPosition(0, 3));
```
Comment (CaeNGnp) [inline SimpCityTests/GameTests.cs]: 0,2 **

**Review by CaeNGnp — CHANGES_REQUESTED** [review]

**Review by CaeNGnp — APPROVED** [review]
Lgtm!

**Review by JSQJ — APPROVED** [review]
the scoring for shop works

**Pull request merged**